lib/graphql/index.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+/**
+ * Singleton object for graphql things.
+ */
+
+const graphql = require('graphql');
```

Comment:
I'm curious how tests are passing, yet graphql is not in the package.json.

What's the use case for adding graphql as a dependency?